### PR TITLE
Update examples so they work, add info to READMEs to allow customization, remove vulns

### PR DESCRIPTION
**Pull request by drbrady8800**
Review Level: Normal
Addresses: DEV-765

- Went through all the examples and made sure they worked, made changes where necessary
- Updated the READMEs to address tje areas where I got tripped up. 
- Added instructions to use their own workspace IDs / live domains.
- Removed the README in `fastapi/frontend` since it wasn't relevant to that example.
- Removed a lot of the react README because it didn't explain the example, but rather how to recreate the example. I linked to the docs for this.
- I removed vulnerabilities for all the examples except:
  - react-native: has circular dependencies so it wasn't possible to resolve all vulnerabilities, the two that are left are:
    1. The ip one that Ryan found for DDG and isn't an issue for us.
    2. One for a package called semver that is moderate and aslo doesn't seem to affect us.
  - react: vulns are in `react-script` package but "fixing" it cause more vulns to pop up.

Most of the changes in the flutter examples were automatic FWIW.

### CODE REVIEW

File: fastapi/README.md [hunk 1]
```diff
@@ -10,14 +10,22 @@ _Note: The included Dockerfiles are not ready for production. See the appropriat
 
 1. Have a [Userfront](https://userfront.com/) account
 1. Have docker installed
-1. Have the following set as environment variables:
```
Comment (stephencattaneo) [inline fastapi/README.md:13]: TBH I'm not sure why this change was made, this feels like a personal preference.

File: php/README.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+brew install php
+```
+2. Install dependencies
+```sh
+brew install composer
+composer install
+```
+3. Run local php server
+```sh
+php -S localhost:3000
```
Comment (stephencattaneo) [inline php/README.md:21]: 4. Question the life choices that led you to using PHP on purpose.

File: fastapi/README.md [hunk 3]
```diff
@@ -10,14 +10,22 @@ _Note: The included Dockerfiles are not ready for production. See the appropriat
...
-        > export USERFRONT_ACCOUNT_ID=...
+    1. NEXT_PUBLIC_USERFRONT_WORKSPACE_ID (Can be found at https://userfront.com/dashboard/workspaces )
 
 ## Running
 
 1. bring up the frontend (Nextjs) and backend (FastAPI) containers
     > docker compose up
 2. visit [http://localhost:3000](http://localhost:3000)
+
+### Issue accessing localhost with Chrome
```
Comment (stephencattaneo) [inline fastapi/README.md:24]: TIL. WTF Google

File: fastapi/README.md [hunk 4]
```diff
@@ -10,14 +10,22 @@ _Note: The included Dockerfiles are not ready for production. See the appropriat
 
 1. Have a [Userfront](https://userfront.com/) account
 1. Have docker installed
-1. Have the following set as environment variables:
+1. Copy .env.example into a file called .env.local
+1. Have the following set as environment variables in .env.local:
     1. USERFRONT_PUBLIC_KEY (Can be found at https://userfront.com/dashboard/jwt )
-        > export USERFRONT_PUBLIC_KEY="-----BEGIN PUBLIC KEY----- ... -----END PUBLIC KEY-----"
```
Comment (stephencattaneo) [inline fastapi/README.md:15]: This feels like a change for personal preference.

File: react/src/App.js [hunk 5]
```diff
@@ -7,9 +7,9 @@ import {
   Navigate,
   useLocation,
 } from "react-router-dom";
-import Userfront, { SignupForm, LoginForm, PasswordResetForm } from "@userfront/toolkit/react";
+import Userfront, { SignupForm, LoginForm, PasswordResetForm, LogoutButton } from "@userfront/toolkit/react";
 
-Userfront.init("demo1234");
+Userfront.init("rbv549xn");
```
Comment (stephencattaneo) [inline react/src/App.js]: Theres a lot of inconsistency in the examples for what what to do with tenantId.  

IMO each of the examples should push the user to the dashboard and use their legit tenantId. 1) that gives the user an idea of what is going on for actually applications 2) provides us a way to gather data for "auth'd user in live/test mode"

File: react-native/package.json [hunk 6]
```diff
@@ -1,21 +1,22 @@
 {
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
-    "web": "expo start --web",
-    "eject": "expo eject"
+    "start": "npx expo start",
+    "android": "npx expo start --android",
```
Comment (stephencattaneo) [inline react-native/package.json:5]: nice. Telling users to install anything globally is probably a bad idea.

File: react/src/App.js (hunk 5 above)
Comment (drbrady8800) [reply react/src/App.js]: I agree but not all of these examples are built to be used with a tenantID other than the ones coded into them.
- AWS API Gateway -> No tenantId necessary, README points to JWT in dashboard
- FastAPI -> Points to JWT and workspaceId in dashboard
- Flutter -> Points to workspaceId in the dashboard
- NextJS -> Not built to function with anything other than demo (it mocks the API)
- PHP -> Relies on the old pre-built forms so only works for the tenantId in the code (their toolIds are hard coded)
- React -> Points to workspaceId in the dashboard
- React Native -> Points to workspaceId in the dashboard
- Vue -> Points to workspaceId in the dashboard

I did accidentally leave in my workspaceId here from testing.

File: fastapi/README.md (hunk 4 above)
Comment (drbrady8800) [reply fastapi/README.md:15]: Since this is started with `docker compose up` wouldn't the original instructions not pass the env variables in?

**Review by RJFelix — APPROVED** [review]
Looking great, thank you a bunch!
